tools/webpack/configs/common.mjs

```diff
@@ -59,9 +59,12 @@ export default (env, asyncChunkName) => {
         namespace: `NRBA-${env.VERSION}.${env.SUBVERSION}`,
         filename: '[file].map[query]',
         moduleFilenameTemplate: 'nr-browser-agent://[namespace]/[resource-path]?[loaders]',
-        publicPath: env.PUBLIC_PATH,
+        publicPath: env.PUBLIC_PATH, // -- leaving source maps on CDN only as they're not strictly necessary
         append: env.SUBVERSION === 'PROD' ? false : '//# sourceMappingURL=[url]'
       }),
+      new webpack.DefinePlugin({ // passing default asset path as string to agent
+        WEBP_CONST_PUBLIC_PATH: JSON.stringify(env.PUBLIC_PATH)
+      }),
```

Comment:
In my local testing, we don't actually need these changes in the webpack config and we can reduce the public-path.js file down to just setting the global `__webpack_public_path__`. This way, webpack is set with the default public path and it only gets overridden if the function is called to set it.

Reply:
Proxy e2e still pass after reverting change to this file 👍 . Thanks for finding out